src/evo/deterministicmns.h

```diff
@@ -169,7 +174,7 @@ class CDeterministicMNList
 {
 public:
     typedef immer::map<uint256, CDeterministicMNCPtr> MnMap;
-    typedef immer::map<uint256, std::pair<uint256, uint32_t>> MnUniquePropertyMap;
+    typedef immer::map<uint256, std::pair<uint256, uint32_t> > MnUniquePropertyMap;
```

Comment:
any chance to lift this rule in the future? That's some historic stuff due to old compilers not being able to properly differentiate between a single `>` and the `>>` operator. @UdjinM6 @nmarley

Reply:
If our modern C++14 (and up) dev env supports this non-spaced format, then yeah, let's remove this rule and move forward (e.g. no reason to keep legacy stuff). I didn't realize this was possible now and thought the space was still required.

Reply:
Imo the `> >` formatting reads better

Reply:
I actually agree with @PastaPastaPasta and prefer the `> >` version personally 🙈

Reply:
Old way is fine then, don't feel strongly on this. Might be easier to parse visually. 🤷‍♂️